tests/_ast/test_sql_visitor.py

```diff
@@ -421,3 +422,133 @@ def test_find_created_database() -> None:
 )
 def test_find_sql_defs_duckdb_not_available() -> None:
     assert find_sql_defs("CREATE TABLE test (id INT);") == SQLDefs([], [], [])
+
+
+@pytest.mark.skipif(not HAS_DUCKDB, reason="Missing DuckDB")
+class TestFindSQLRefs:
+    @staticmethod
+    def test_find_sql_refs_simple() -> None:
+        sql = "SELECT * FROM test_table;"
+        assert find_sql_refs(sql) == ["test_table"]
+
+    @staticmethod
+    def test_find_sql_refs_multiple() -> None:
+        sql = """
+        SELECT * FROM table1;
+        SELECT * FROM table2;
+        """
+        assert find_sql_refs(sql) == ["table1", "table2"]
+
+    @staticmethod
+    def test_find_sql_refs_with_schema() -> None:
+        sql = "SELECT * FROM my_schema.my_table;"
+        assert find_sql_refs(sql) == ["my_schema", "my_table"]
```

Comment:
Why does `my_table` need to be a ref here?  Couldn't the ref be just the schema (or catalog)?

Reply:
Couldn't you have `my_table` belonging to two different schemas though?

`my_first_schema.my_table`

and

`my_second_schema.my_table`

Creating `my_table` in the second schema shouldn't cause cells referring to `my_table` in the first schema to run, right?

Reply:
yea ideally not, but right now we don't have any sort of inheritance/scope with variables (just global or not)

Reply:
to me, the lesser of two evils is the extra dependency rather than no dependency (the app/notebook would fail on startup in this case.)